composeApp/src/commonMain/kotlin/presentation/EventViewModel.kt

```diff
@@ -0,0 +1,4 @@
+package presentation
+
+class EventViewModel {
```

Comment:
There's an interesting article about how to write an implementation of a ViewModel to support configuration changes in a Compose Multiplatform project

https://blog.stackademic.com/viewmodel-management-with-jetpack-compose-6951906486e6

Reply:
Quick update: I tried Recompose in another project (to implement ViewModels and Navigation), works really well!